src/lib.rs

```diff
@@ -206,6 +206,7 @@ impl<T> Counter for T
 /// = 2048 * 2^(k-1)`, which is the k-1'th bucket's end. So, we would use the previous bucket
 /// for those lower values as it has better precision.
 ///
+#[derive(Debug)]
```

Comment:
Ah yes, an enum for the errors is on my to-do list. The string matching makes me sad, but I'd rather figure out the `new_*` refactoring first (and figure out what the error cases are).